tests/Mouf/Database/TDBM/TDBMDaoGeneratorTest.php

```diff
@@ -36,25 +30,35 @@
  */
 class TDBMDaoGeneratorTest extends TDBMAbsctractServiceTest {
 
-    /** @var TDBMDaoGenerator $tdbmDaoGenerator */
-    protected $tdbmDaoGenerator;
-
-    protected function setUp() {
-        $this->tdbmDaoGenerator = new TDBMDaoGenerator($this->dbConnection);
-    }
-
-    /**
-     * @depends testFake
-     */
 	public function testDaoGeneration() {
-        $daoFactoryClassName = "daoFactory";
-        $sourcedirectory = "Mouf/Database/TDBM/Dao";
-        $daonamespace = "Mouf/Database/TDBM/Dao/";
-        $beannamespace = "Mouf/Database/TDBM/Dao/Bean/";
+        $daoFactoryClassName = "DaoFactory";
+        // Put your right directory from your home
+        // Todo, find a good way to do it
+        $sourcedirectory = "projects/mouf/database.tdbm/tests/Mouf/Database/TDBM";
```

Comment:
Yes! :)
Create a directory /tests/generated in the package (put an empty `.gitkeep` file in it so you can commit it).
Then, refer to this directory using the __DIR__ constant.

For instance:
`$sourceDirectory = __DIR__.'/../../../../../generated';` 

=> Note: count the number of ".." correctly, it is the hardest part.